assets/js/modules/reader-revenue-manager/components/dashboard/PublicationApprovedOverlayNotification.js

```diff
@@ -0,0 +1,139 @@
+import {
+	MODULES_READER_REVENUE_MANAGER,
+	UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION,
+} from '../../datastore/constants';
+
+export const RRM_PUBLICATION_APPROVED_OVERLAY_NOTIFICATION =
+	'rrmPublicationApprovedOverlayNotification';
+
+export default function PublicationApprovedOverlayNotification() {
+	const isViewOnly = useViewOnly();
+	const dashboardType = useDashboardType();
+
+	const isDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isItemDismissed(
+			RRM_PUBLICATION_APPROVED_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const { dismissOverlayNotification } = useDispatch( CORE_UI );
+	const serviceURL = useSelect( ( select ) =>
+		select( MODULES_READER_REVENUE_MANAGER ).getServiceURL()
+	);
+
+	const showApprovedNotificationUI = useSelect( ( select ) =>
+		select( CORE_UI ).getValue(
+			UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION
+		)
+	);
+
+	/**
+	 * Determines whether the publication approved notification should be shown when the following conditions are met.
+	 * - The notification has not been dismissed.
+	 * - The notification UI is enabled.
+	 * - The user is not in view-only mode.
+	 * - The user is on the main dashboard.
+	 */
+	const shouldShowNotification =
+		isDismissed === false &&
+		showApprovedNotificationUI === true &&
+		! isViewOnly &&
+		dashboardType === VIEW_CONTEXT_MAIN_DASHBOARD;
+
+	const isDismissing = useSelect( ( select ) =>
+		select( CORE_USER ).isDismissingItem(
+			RRM_PUBLICATION_APPROVED_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const dismissNotification = () => {
+		// Dismiss the notification, which also dismisses it from
+		// the current user's profile with the `dismissItem` action.
+		dismissOverlayNotification(
+			RRM_PUBLICATION_APPROVED_OVERLAY_NOTIFICATION
+		);
+	};
+
+	if ( isDismissing ) {
+		return null;
+	}
+
+	return (
+		<OverlayNotification
+			className="googlesitekit-reader-revenue-manager-publication-approved-notification"
+			GraphicDesktop={ ReaderRevenueManagerIntroductoryGraphicDesktop }
+			GraphicMobile={ ReaderRevenueManagerIntroductoryGraphicMobile }
+			shouldShowNotification={ shouldShowNotification }
+			notificationID={ RRM_PUBLICATION_APPROVED_OVERLAY_NOTIFICATION }
+		>
+			<div className="googlesitekit-overlay-notification__body">
+				<h3>
+					{ __(
+						'Your Reader Revenue Manager publication is approved',
+						'google-site-kit'
+					) }
+				</h3>
+				<p>
+					{ __(
+						'Unlock your full reader opportunity by enabling features like subscriptions, contributions and newsletter sign ups',
+						'google-site-kit'
+					) }
+				</p>
+			</div>
+			<div className="googlesitekit-overlay-notification__actions">
+				<Button
+					tertiary
+					disabled={ isDismissing }
+					onClick={ dismissNotification }
+				>
+					{ __( 'Maybe later', 'google-site-kit' ) }
+				</Button>
+
+				<Button
+					href={ serviceURL }
+					trailingIcon={ <ExternalIcon width={ 14 } height={ 14 } /> }
```

Comment:
According to the Figma designs, I believe the dimensions here should be `13`.

```suggestion
					trailingIcon={ <ExternalIcon width={ 13 } height={ 13 } /> }
```